### PR TITLE
[loki-distributed] upgraded the podManagementPolicy to Parallel and bumped the chart major version

### PR DESCRIPTION
**Pull request by sherifkayad**
upgraded the podManagementPolicy to Parallel and bumped the chart major version. This PR addresses #435 

Signed-off-by: Sherif Ayad <sherif.k.ayad@gmail.com>

### CODE REVIEW

File: charts/loki-distributed/Chart.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: loki-distributed
 description: Helm chart for Grafana Loki in microservices mode
 type: application
 appVersion: 2.2.0
-version: 0.30.0
+version: 1.0.0
```
Comment (Whyeasy) [inline charts/loki-distributed/Chart.yaml:6]: No need to use 1.0.0, moving to 0.31.0 is already considered a major release with helm charts.

File: charts/loki-distributed/README.md [hunk 2]
```diff
@@ -18,6 +18,22 @@ Add the following repo to use the chart:
 helm repo add grafana https://grafana.github.io/helm-charts
 ```
 
+## Upgrading
+
+### Upgrading an existing Release to a new major version
+
+A major chart version change (like v0.0.30 -> v1.0.0) indicates that there is an incompatible breaking change needing manual actions.
```
Comment (Whyeasy) [inline charts/loki-distributed/README.md:25]: Small typo with the versions.  Same story here, no need to use v1.0.0.

File: charts/loki-distributed/README.md [hunk 3]
```diff
@@ -18,6 +18,22 @@ Add the following repo to use the chart:
 helm repo add grafana https://grafana.github.io/helm-charts
 ```
 
+## Upgrading
```
Comment (Whyeasy) [inline charts/loki-distributed/README.md:21]: Please update the docs via the `README.md.gotmpl` and `helm-docs` .

**Review by Whyeasy — CHANGES_REQUESTED** [review]

**Comment by sherifkayad** [comment]
@Whyeasy All review comments are resolved now :) .. Could you please re-check?

**Comment by sherifkayad** [comment]
@Whyeasy Any idea why the Lint Code Base check is failing? .. I ran helm-docs and committed but still getting the error

**Comment by Whyeasy** [comment]
@sherifkayad I think the formatting got a bit messed up. For example: https://github.com/grafana/helm-charts/pull/448/files#diff-e25e5957ac51b5ae427b4f62ace7e1fd562428c3db6aa8f8bc286eb4b0adbf03R14

Maybe because of some formatter in your IDE 😄 . If you look at the linting you see the  `-` at those lines.

**Comment by sherifkayad** [comment]
@Whyeasy I can't see to be able to fix it 😢 .. Can you elaborate more? .. I am using VSCode on Windows and I guess that could be the root of all evil here

**Comment by sherifkayad** [comment]
@Whyeasy I committed something in attempt to fixing the issue .. let's hope it passes

**Comment by Whyeasy** [comment]
@sherifkayad I ran helm-docs for you, seems to fix the issue.

FYI: Take a look in to WSL2 + VsCode when you are on Windows.

**Review by Whyeasy — APPROVED** [review]

**Comment by sherifkayad** [comment]
@Whyeasy Thanks a lot for your help. Happy to hear it's now fixed 🚀

**Pull request merged**